test/testcafe/spec/DeviceChallengePollView_spec.js

```diff
@@ -317,6 +405,32 @@ test
     await t.expect(identityPage.getIdentifierValue()).eql('Test Identifier');
   });
 
+test
+  .requestHooks(loopbackPollMockLogger, loopbackPollFailedMock)('next poll should not start if previous is failed', async t => {
+    loopbackPollMockLogger.clear();
+    await setup(t);
+    await wait(10_000);
```

Comment:
Are we able to wait explicit for UI changes as other tests do?

Reply:
Can you please link any example, to be sure that we are on the same page?

Reply:
actually for this case, would `t.wait` work for you?
https://github.com/okta/okta-signin-widget/blob/master/test/testcafe/spec/EnrollAuthenticatorEmail_spec.js#L108

Reply:
updated